flatgfa/src/ops/gaf.rs

```diff
@@ -191,6 +192,46 @@ impl ChunkEvent {
             ChunkRange::None => {}
         }
     }
+    pub fn get_seq_string(&self, gfa: &flatgfa::FlatGFA) -> String {
+        let seq = gfa.get_seq_oriented(self.handle);
+    
+        match self.range {
+            ChunkRange::Partial(start, end) => seq.slice(start..end).to_string(),
+            ChunkRange::All => seq.to_string(),
+            ChunkRange::None => String::new(),
+        }
+    }
```

Comment:
This duplicates some code with the `print_seq` method. Please deduplicate this so we only have one copy of the `match`… perhaps it would be most straightforward to make `print_seq` call this instead of doing its own `match`? Or maybe we want a third method they both call that returns a `Sequence`?